src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitMqQueueMessageContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CoreWCF.Queue.Common;
+
+namespace CoreWCF.Channels
+{
+    public class RabbitMqQueueMessageContext : QueueMessageContext
+    {
+        /// <summary>
+        /// Delivery confirmation number for a message read from a RabbitMQ queue
+        /// </summary>
+        public ulong DeliveryTag { get; set; }
+    }
```

Comment:
Is ulong necessary? Are the numbers ever > int64.MaxValue? The reason I ask is ulong isn't cls compliant which means there are some .NET languages which can't use this api. Generally using non-cls compliant types should be avoided if possible. If it really does need an unsigned 64 bit number, then it's okay.